visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-console */
+/* eslint-disable no-restricted-globals */
+import path from 'path';
+import execa from 'execa';
+import fse from 'fs-extra';
+// eslint-disable-next-line import/extensions
+import { baseBranch, wrappers } from './config.mjs';
+
+const currentBranch = process?.env?.CURRENT_BRANCH;
+const dirs = {
+  init: process.cwd(),
+  examples: '../examples/next/visual-tests',
+  codeToRun: 'demo',
+  screenshots: './tests/screenshots',
+};
+
+process.chdir(dirs.examples);
+execa.command('npm install');
+
+// If we are on a base branch, we do not want to run all of tests
+// and make screenshots for all of wrappers.
+// On base branch we need to create golden screenshots from main wrapper only,
+// which is declared as a first position in wrappers array.
+
+for (let i = 0, maxi = (currentBranch === baseBranch ? 1 : wrappers.length); i < maxi; ++i) {
+  process.chdir(`${wrappers[i]}`);
+  // eslint-disable-next-line no-await-in-loop
+  await execa.command('npm install', { stdout: 'inherit' });
```

Comment:
```suggestion
  // eslint-disable-next-line no-await-in-loop
  await execa.command('npm install', { stdout: 'inherit', cwd: wrappers[i] });
```
IMO using `chdir` in that case is not the best idea. Changing the directory multiple times within the same script can easily lead to errors. As the changing directory needs to be tracked. That's why I'd propose changing all  `await execa.command(...` calls to use `cwd` property. Thanks to that you can remove all `chdir` calls from the code.

Reply:
Done.

Reply:
Done.